### ISSUE TITLE
Stop making strange transforms when loading images

### ISSUE THREAD

**Issue by Abe404** [Abe404/RootPainter3D#29]
See: 
https://github.com/Abe404/RootPainter3D/blob/2d3155d7009fa2f49ff6868253960bb6f366895b/painter/src/im_utils.py#L56

Consider instead working with the data in the original format.

**Comment by Abe404** [comment]
https://github.com/Abe404/RootPainter3D/blob/bedb9dc66d0c7d8cb9bfde2d0b4c2eedfd99cd6a/trainer/im_utils.py#L183

**Comment by Abe404** [comment]
This has been addressed in https://github.com/Abe404/RootPainter3D/commit/ec8bc6323664152aff1905ed759d085e786d3bdf in the trainer code.

It's  commit on a different branch. I will close this issue once it is merged into master and it has been tested (including with the client, and the necessary modifications made in the client).

**Comment by Abe404** [comment]
Fixed but things looks a bit different in the client. I will address that as a separate issue. I think we need to look at using the affine translation matrix to change how the images appear.
